tests/parsers/apache_access.py

```diff
@@ -100,5 +100,37 @@ def testParse(self):
     self.assertEqual(warning.parser_chain, 'apache_access')
 
 
+    # Test vhost_combined log format event.
+    event = events[9]
+    self.CheckTimestamp(event.timestamp, '2018-01-13 19:31:17.000000')
+
+    self.assertEqual(event.ip_address, '192.168.0.2')
+    self.assertEqual(event.remote_name, '-')
+    self.assertEqual(event.user_name, '-')
+
+    self.assertEqual(
+        event.http_request,
+        'GET /wp-content/themes/darkmode/evil.php HTTP/1.1')
+
+    self.assertEqual(event.http_response_code, 200)
+    self.assertEqual(event.http_response_bytes, 1063)
+
+    expected_message = (
+        'http_request: GET /wp-content/themes/darkmode/evil.php HTTP/1.1 '
+        'from: 192.168.0.2 '
+        'code: 200 '
+        'referer: - '
+        'user_agent: Mozilla/5.0 (Windows NT 7.1) AppleWebKit/534.30 '
+        '(KHTML, like Gecko) Chrome/12.0.742.112 Safari/534.30 '
+        'server_name: plaso.log2timeline.net '
+        'port: 443'
+    )
```

Comment:
move to end of previous line

Reply:
This is consistent with the rest of the file, so I'll update the whole file.